tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+import os
+from io import BytesIO
+
+from django.core.files import File
+
+import pytest
+from PIL import Image
+
+
+@pytest.fixture
+def text_file(request):
+    PATH = 'dummy.txt'
+    with open(PATH, 'w') as tmp:
+        tmp.write('some additional information')
+
+    def fin():
+        os.remove(PATH)
+
+    request.addfinalizer(fin)
+    return open(PATH, 'rb')
```

Comment:
doesn't this leak file descriptors? Might be better to assign the return value of `open(...)` to a variable, add the `.close()` call on that to the finalizer, and then return the opened file.